index.html

```diff
@@ -159,6 +159,7 @@ <h3 id="out-of-scope">Out of Scope</h3>
             <ul class="out-of-scope">
               <li>The mandate of any specific style of supporting infrastructure (such as a DLT) for a verifiable credentials ecosystem</li>
               <li>The specification of new cryptographic primitives</li>
+              <li>The specification of APIs or protocols (although implications for APIs and protocols of data model and data integrity choices made MAY be discussed)</li>
```

Comment:
```suggestion
              <li>The specification of APIs or protocols.</li>
```

Reply:
we don't need the although here, though i appreciate the attempt to give the WG permission to discuss thing in a parenthetical on an out of scope item, i think its better addressed in direct items with a clear track (WG-NOTE vs TR).